packages/components/src/styles/components/dropdown.scss

```
@@ -136,6 +136,12 @@ $hds-dropdown-toggle-border-radius: $hds-button-border-radius;
 
 .hds-dropdown-toggle-button--size-small {
   padding-right: 0.375rem;
+
+  // For small variant with chevron, force the icon size to custom (even if the SGV size is `16px`)
+  .hds-icon {
```

It would be wiser for us to limit this override to the toggle chevron, otherwise it will affect any icon in the dropdown toggle (e.g. the `@icon`, an icon in the badge, both currently set at `0.75rem`, which coincidentally translates to `12px`, but we can't rely on that from a systems perspective).
```suggestion
  .hds-dropdown-toggle-chevron .hds-icon {
```

---

My bad I missed this, I was tricked by the fact that the parent selector `.hds-dropdown-toggle-button--size-small` so I thought it was safe to use just `.hds-icon`, but I forgot that the Dropdown toggle looks like a Button but is not a Button, and can contain more things 🤦‍♂️

(that said, the badge is always small (so the badge icon is `12px` anyway) and the toggle icon is also `12px` when small, so visually nothing changed, and that's why probably the VRT tests didn't fail)

---

yes not a big deal, really. a fix was added in #2533